source/messaging/channels-basics.rst

```diff
@@ -18,29 +18,65 @@ Channels Basics
   :target: https://mattermost.com/deploy
   :alt: Available for Mattermost Self-Hosted deployments.
 
-**Write messages** using the text input box at the bottom of the screen. Press ENTER to send a message. Use SHIFT+ENTER to create a new line without sending a message. To send messages on CTRL+ENTER and use ENTER to insert new lines go to **Settings > Advanced > Send messages on CTRL/CMD+ENTER**.
+Write messages
+--------------
 
-**Reply to messages** by clicking the reply arrow next to the message text.
+Write messages using the text input box at the bottom of the screen. Press ENTER to send a message. Use SHIFT+ENTER to create a new line without sending a message. 
```

Comment:
If the user has set their input to be CMD+Enter to post, should we mention that? In that case they just use ENTER for a new line (and possibly SHIFT+ENTER, I have to test that).